pcsx2/GS/GSdx.def

```diff
@@ -1,4 +1,4 @@
-; GSdx.def : Declares the module parameters for the DLL.
+; GS.def : Declares the module parameters for the DLL.
```

Comment:
Forgotten removal of .def file?

Reply:
not forgotten, I'm waiting for the windows port to do that, noted it in my todo list above :)

Reply:
All right.